README.md

```diff
@@ -48,6 +48,57 @@ required go version: 1.12
 go get github.com/jesseduffield/lazydocker
 ```
 
+### Docker
+
+1. <details><summary>Build it...</summary><p>
+
+    - If you have a x86_64 CPU architecture
+
+        ```sh
+        docker build -t lazydocker . https://github.com/jesseduffield/lazydocker.git
+        ```
+
+    - If you have a ARM 32 bit v6 architecture
+
+        ```sh
+        docker build -t lazydocker \
+        --build-arg BASE_IMAGE_BUILDER=arm32v6/golang \
+        --build-arg BASE_IMAGE=arm32v6/alpine \
+        --build-arg GOARCH=arm \
+        --build-arg GOARM=6 \
+        https://github.com/jesseduffield/lazydocker.git
+        ```
+
+    - If you have a ARM 32 bit v7 architecture
+
+        ```sh
+        docker build -t lazydocker \
+        --build-arg BASE_IMAGE_BUILDER=arm32v7/golang \
+        --build-arg BASE_IMAGE=arm32v7/alpine \
+        --build-arg GOARCH=arm \
+        --build-arg GOARM=7 \
+        https://github.com/jesseduffield/lazydocker.git
+        ```
+
+    - If you have a ARM 64 bit v8 architecture
+
+        ```sh
+        docker build -t lazydocker \
+        --build-arg BASE_IMAGE_BUILDER=arm64v8/golang \
+        --build-arg BASE_IMAGE=arm64v8/alpine \
+        --build-arg GOARCH=arm64 \
+        https://github.com/jesseduffield/lazydocker.git
+        ```
+
+    </p></details>
+1. Run it
+
+    ```sh
+    docker run -it -v /var/run/docker.sock:/var/run/docker.sock lazydocker
```

Comment:
Maybe we can also provide an option to mount config file?

Reply:
Yes good idea, I think it's in the `/.config` directory from what I've seen? Or can we specify its location?

Reply:
`$HOME/.config/jesseduffield/lazydocker/config.yml`

Reply:
We could use a volume as well so there is persistence without having to bind mount, but that's always a bit confusing on how to use for newcomers too. What do you think?

Reply:
`$HOME/.config/jesseduffield/lazydocker` would be best, I think.